projects/arcan/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+{
+  # NOTE:
+  # - Check `projects/models.nix` for a more detailed project structure
+  # - Each program/service must have at least one example
+  # - Set attributes to `null` to indicate that they're needed, but not available
+  metadata = {
+    summary = "Explorative p2p protocol for fast and secure remote desktops";
+    subgrants = [
+      "arcan-A12"
+      "arcan-A12-directory"
+      "arcan-A12-tools"
+    ];
+  };
+
+  nixos.modules.programs = {
+    arcan = {
+      name = "arcan";
+      module = ./module.nix;
+      examples.base = {
+        module = ./example.nix;
+        description = "testing documentation";
+        tests.basic = null;
+      };
+      links = {
+        build = {
+          text = "arcan Documentation";
+          url = "https://github.com/letoram/arcan#compiling";
+        };
+        test = {
+          text = "arcan Documentation";
+          url = "https://github.com/letoram/arcan/tree/master/tests";
+        };
+      };
+    };
+
+    # needed, but not available
+    foobar-cli = null;
+  };
+
+  # NOTE: same structure as programs
+  nixos.modules.services = {
+  };
```

Comment:
The [new template](https://github.com/ngi-nix/ngipkgs/tree/a60bfecaec243afe8af042ef061a0de3594f8025/templates/project) is more descriptive. Please update your main branch and use that next time.

```suggestion
```